ksql-engine/src/main/java/io/confluent/ksql/util/KafkaTopicClientImpl.java

```diff
@@ -129,6 +137,12 @@ public boolean isTopicExists(final String topic) {
     }
   }
 
+  @Override
+  public DescribeConfigsResult describeConfigs(String topicName) {
+    return adminClient.describeConfigs(
```

Comment:
This should use the `RetryHelper.executeWithRetries` method. Otherwise the command will fail for even retriable exceptions.

Reply:
Changed the method to use RetryHelper and return the clean up policy only.